boilerplate/app.config.ts

```diff
@@ -34,6 +34,10 @@ module.exports = ({ config }: ConfigContext): Partial<ExpoConfig> => {
         ],
       },
     },
-    plugins: [...existingPlugins, require("./plugins/withSplashScreen").withSplashScreen],
+    plugins: [
+      ...existingPlugins,
+      require("./plugins/withSplashScreen").withSplashScreen,
+      "expo-build-properties",
```

Comment:
I think we can move `expo-build-properites` to the static config, `app.json` next to all the other ones.

This area is somewhat reserved for our inline plugins.

Which I'm torn about in another discussion, because I think this `withSplashScreen` plugin can go away. But it also provides an example on how to do an inline TypeScript plugin in Ignite.

Perhaps we need to move it to a recipe in a follow up PR